.github/workflows/release.yml

```diff
@@ -3,15 +3,21 @@ name: release
 on:
   workflow_dispatch:
     inputs:
-      tag:
-        description: "Git tag (leave empty for dry run)"
+      ref:
+        description: "Git tag or branch (leave empty for dry run)"
         required: false
+      prerelease:
+        description: "Is this a pre-release?"
+        required: true
+        type: boolean
 
 jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
     environment: gpg
+    env:
+      TRIGGER_PRERELEASE_FLOW: ${{ inputs.ref == 'main' && inputs.prerelease }}
```

Comment:
I would change the logic here slightly. It should be hard to do a non-prerelease by accident. Thus, I would suggest that it's a prerelase if it's on the main branch **or** input.prerelease is true.

One should not be able to make a non-prerelease release on the main branch. And if someone specifies inputs.prerelease as true, then it should also definitely not be a full release.

While someone specifying inputs.prerelease on a branch like v1.6 **should** create a **prerelease**.

I think this is just a case of changing this to `${{ inputs.ref == 'main' || inputs.prerelease }}`

Reply:
Done.